pandas/core/reshape/reshape.py

```diff
@@ -751,6 +751,138 @@ def _convert_level_number(level_num, columns):
     return result
 
 
+def from_dummies(data, columns=None, prefix_sep="_", dtype="category", fill_first=None):
+    """
+    The inverse transformation of ``pandas.get_dummies``.
+
+    Parameters
+    ----------
+    data : DataFrame
+    columns : list-like, default None
+        Column names in the DataFrame to be decoded.
+        If `columns` is None then all the columns will be converted.
+    prefix_sep : str, default '_'
+        Separator between original column name and dummy variable
+    dtype : dtype, default 'category'
+        Data dtype for new columns - only a single data type is allowed
+    fill_first : str, list, or dict, default None
```

Comment:
I think I'm -1 on this feature; not sure I see it being super useful outside of niche applications

Reply:
@WillAyd thanks for your feedback - when you say "this feature", do you mean `from_dummies`, or just `fill_first`?

Reply:
fill_first